app/controllers/unattended_controller.rb

```diff
@@ -249,7 +249,7 @@ def safe_render(template)
     end
 
     begin
-      render :inline => "<%= unattended_render(@unsafe_template_content, @template_name).html_safe %>"
+      render :plain => unattended_render(@unsafe_template_content, @template_name)
```

Comment:
This breaks my provisioning, are you able to get a kickstart template with this change in @timogoebel @lzap ?

Reply:
Yeah filed https://github.com/theforeman/foreman/pull/5258 let's discuss there.